drivers/interrupt_controller/intc_ite_it8xxx2.c

```diff
@@ -250,6 +250,31 @@ static void intc_irq0_handler(const void *arg)
 
 void soc_interrupt_init(void)
 {
+#ifdef CONFIG_ZTEST
```

Comment:
Please add a comment explaining the reason why this code path asserts the watchdog reset.

Reply:
Ok, add a comment, done.